pkg/client.go

```diff
@@ -105,7 +105,8 @@ func (c *Client) AllMonitors() ([]Monitor, error) {
 	offset := 0
 	r := Response{}
 	for offset <= r.Pagination.Total {
-		data := []byte(fmt.Sprintf("{\"offset\": \"%d\", \"limit\": \"%d\"}", offset, maxRecordsPerRequest))
+		data := []byte(fmt.Sprintf("{\"offset\": \"%d\", \"limit\": \"%d\", \"alert_contacts\": \"1\"}", offset, maxRecordsPerRequest))
+		//data := []byte(fmt.Sprintf("{\"offset\": \"%d\", \"limit\": \"%d\"}", offset, maxRecordsPerRequest))
```

Comment:
Any reason why this line is commented out?

Reply:
No reasons. There was previous code. Deleted this string.